applications/crato.md

```diff
@@ -0,0 +1,127 @@
+# Crust Grant Proposal
+
+* **Project Name:** Crato
+* **Team Name:** Nash Cloud
+* **Payment Address:** 0xfE219465caA035D568BB4d9F2263cde3cf279AD8
+* **Receivable Token:** CRU
+
+> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+> ⚠️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
+
+### Overview
+
+Please provide the following:
+  * A project name: Crato
+  * A brief description of your project: Sophisticated Pin Service Built On IPFS
+  * An indication of how your project relates to / integrates into Crust: Crato seriously takes its role as eco-partner of Crust to
+provide decentralized meta data management service with easy
+IPFS API and toolkit.
+  * An indication of why your team is interested in creating this project: We have always been interested in field of Storage, by the power of IPFS and Crust blockchain, we have chance to creat a brand new storage service.
+
+### Project Details 
```

Comment:
Missing key information, please fill in it completely

Reply:
Already updated at your request.